UOP1_Project/Packages/manifest.json

```diff
@@ -1,16 +1,17 @@
 {
   "dependencies": {
-    "com.unity.addressables": "1.8.4",
+    "com.unity.addressables": "1.8.5",
     "com.unity.cinemachine": "2.6.0",
-    "com.unity.ide.rider": "1.1.4",
+    "com.unity.ide.rider": "1.2.1",
+    "com.unity.ide.visualstudio": "2.0.2",
     "com.unity.ide.vscode": "1.2.1",
     "com.unity.inputsystem": "1.0.0",
-    "com.unity.polybrush": "1.0.1",
+    "com.unity.polybrush": "1.0.2",
     "com.unity.probuilder": "4.2.3",
     "com.unity.progrids": "3.0.3-preview.6",
-    "com.unity.render-pipelines.universal": "7.3.1",
+    "com.unity.render-pipelines.universal": "8.2.0",
     "com.unity.settings-manager": "1.0.3",
-    "com.unity.textmeshpro": "2.0.1",
+    "com.unity.textmeshpro": "3.0.1",
```

Comment:
Please don't update any packages, you can find the contributions guide here:  https://docs.google.com/document/d/1PwBF4yQl69RxvVHZ2m2iiy5pYjd9QO-VcuXWDjB7QwA/edit#heading=h.geg4xz28ebgh